crates/bevy_picking/src/lib.rs

```diff
@@ -72,11 +73,7 @@ pub struct Pickable {
     ///
     /// Entities without the [`Pickable`] component will block by default.
     pub should_block_lower: bool,
-    // TODO: remove this allow after focus/hover is implemented in bevy_picking
-    #[allow(rustdoc::broken_intra_doc_links)]
-    /// Should this entity be added to the [`HoverMap`](focus::HoverMap) and thus emit events when
-    /// targeted?
-    ///
```

Comment:
I think this line was supposed to be part of the documentation and not the TODO to be removed.